### ISSUE TITLE
Add Accessibility Statements to Jurisdictional Portals 

### ISSUE THREAD

**Issue by sarahlazarich** [bloom-housing/bloom#3637]
AC:
- We should have an accessibility statement in the footer of all the jurisdictional portals.

**Comment by sarahlazarich** [comment]
sj will roll into doorway soon + this will be true. we realistically won't do that before
